### DISCUSSION TITLE
Is Windows supported?

### DISCUSSION THREAD

**Discussion by beedell-roke** [KDAB/hotspot#685] [Q&A] [answered]
I'd like to get a flamegraph for https://github.com/microsoftfeedback/WinDbg-Feedback/issues/248#issuecomment-2508995080 to diagnose what it's temporarily hanging at whenever it tries to read a Linux ISO's contents written to a storage device.

**Comment by milianw** [comment, marked as answer]
You can run hotspot in WSL to analyze `perf.data` you recorded on a linux host, but afair `perf record` does not work in WSL. So you probably need to use a different tool to profile on Windows. I can recommend UIforETW+WPA, esp. based on your problem description which may be due to some off-CPU issue.

Alternatively, try VTune or the profiler(s) integrated into Visual Studio.

Good luck with your problem

**Reply by beedell-roke** [reply]
Thank you, @milianw.
